pkg/inject/oauth_proxy_test.go

```diff
@@ -48,10 +62,12 @@ func TestOAuthProxyTLSSecretVolumeIsNotAddedByDefault(t *testing.T) {
 	assert.Len(t, dep.Spec.Template.Spec.Volumes, 0)
 }
 
-func TestOAuthProxyConsistentServiceAccountName(t *testing.T) {
+func TestOpenShiftOAuthProxyConsistentServiceAccountName(t *testing.T) {
 	// see https://github.com/openshift/oauth-proxy/issues/95
 	jaeger := v1.NewJaeger(types.NamespacedName{Name: "my-instance"})
 	jaeger.Spec.Ingress.Security = v1.IngressSecurityOAuthProxy
+	viper.Set("platform", "openshift")
```

Comment:
Most of the tests add this as the first thing in the test, implying that this is set before anything we call ourselves.

Reply:
What do you suggest we do with this?